src/tests/services/domains-services/edit-domain-service.test.js

```diff
@@ -56,12 +56,22 @@ describe('DomainsServices', () => {
       statusCode: 200
     })
     const { sut } = makeSut()
-
     const feedbackMessage = await sut(fixtures.domainMock)
 
     expect(feedbackMessage).toBe('Your domain has been edited')
   })
 
+  it('should return a success message on null digital certificate id', async () => {
```

Comment:
não colocar detalhe de implementação no  título. Colocar texto sem o null ( detalhe de implementação , pois representa um valor fixo, que pode mudar, para por exemplo , undefined), assim como o 'id', nome de propriedade, portanto, um detalhe de implementação.
o texto sem detalhes de implementação ficaria mais voltado pro comportamento:

'should return a success message even when not provided a digital certificate'

Reply:
apenas como referência, esse título que você usou num outro teste:
should correctly parse the returned a domain without digital certificate

representa exatamente um bom título de teste, que não coloca detalhe de implementação no título e descreve bem o comportamento testado.